kong/ci/kong.rake

```diff
@@ -31,7 +31,7 @@ namespace :ci do
       sh %(docker run -d --name #{container_name} --link #{container_name_db}:kong-database \
         -e "KONG_DATABASE=cassandra" -e "KONG_CASSANDRA_CONTACT_POINTS=#{container_name_db}" -e "KONG_PG_HOST=#{container_name_db}" \
         -p #{container_port1}:#{container_port1} -p #{container_port2}:#{container_port2}  -p #{container_port3}:#{container_port3} \
-        -p #{container_port4}:#{container_port4}  -p 7946:7946/udp mashape/kong:0.8.1)
+        -p #{container_port4}:#{container_port4}  -p 7946:7946/udp kong:0.9.0)
```

Comment:
Could you use `kong_version` instead? And update the default version defined above (and the version in `travis.yml`, unless you want to remove it from the travis file)